views/pod.html

```diff
@@ -15,7 +15,7 @@
     <dt>Phase</dt>
     <dd>{{resource.status.phase}}</dd>
     <dt>Node</dt>
-    <dd>{{resource.status.host}}</dd>
+    <dd>{{resource.spec.host}}</dd>
```

Comment:
Once `spec.host` is set, it can never be changed.  It is set via binding (scheduling).  In theory you can create a pod and target exactly where you want it to go, but the normal path is to let the scheduler set it.